### PR TITLE
All game data is now passed to the Game class in the constructor.

### PR DESCRIPTION
**Pull request by ehgoodenough**
This is to make it easier to setup tests; just instantiate a new instance of the Game class, passing it only the game data you're testing. :]

### CODE REVIEW

**Comment by ehgoodenough** [comment]
@Willamin @brandonhutchins @jcpirtle @tpayne1, I'm waiting on one of you to approve this pull request before I merge it. (This is your chance to leave picky comments on MY code as revenge)

**Review by jcpirtle — APPROVED** [review]

**Pull request merged**
